### ISSUE TITLE
"How jQuery Works" suggestions

### ISSUE THREAD

**Issue by nikgervae** [jquery/learn.jquery.com#506]
I just started learning about jQuery yesterday, and this first page was pretty confusing. It wasn't until well into "$ vs. $()", many pages later, that I really understood some fundamental concepts.

Early on in "How jQuery Works", it introduces the ready event with this line of code:

```
$( document ).ready(function() {
```

$ not explained at all, so I had no idea what was going on. This introductory chapter would do well to start instead with jQuery( document ), then explain that $ is a convenience shorthand, which will be used from then on. It should also explain that jQuery aka $ is an object and a function, and that invoking it as a function does particular things—in this case, $(document) returns a jQuery instance wrapping/representing the whole document. (The later page "$ vs $()" also says nothing about $ being just another name for jQuery. I found that out somewhere else, and I don't recall exactly where at the moment.)

Also left implicit here is chaining. It should be repeated/emphasized that the following .ready() call is a function of the wrapped jQuery object and not of the DOM object.

The result of $( "a" ) is also not quite clear. As worded, the documentation sounds like that expression selects the one link in the document, but later in showing .addClass(), the documentation says calling that function affects _all_ links.

To summarize, I'd like to see these fundamental points made in the very beginning:
- jQuery() aka $() wraps DOM objects with rich, cross-brower functionality.
- You can select/wrap DOM objects by direct variable reference, or using powerful string expressions based on HTML tag, id, or class (and more).
- A jQuery selection can wrap a single DOM object, or multiple DOM objects, and functions called on the jQuery selection affect all of the wrapped DOM objects.

Anyhow, I'm well into the rest of the documentation now and very impressed. It's just this very intro that could use a little more up-front explanation. Keep up the good work!
